OK, the snap now builds! Hooray! However, it doesn't work when run. I've built it, and installed with `sudo snap install pick-colour-picker_master_amd64.snap  --dangerous`, but then when I actually _run_ this with `pick-colour-picker`, I get this:

```
$ pick-colour-picker 

** (pick-colour-picker:6507): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-mUvR6E2Yut: Permission denied
/snap/pick-colour-picker/x1/lib/python2.7/site-packages/pick/__main__.py:1: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, Gdk, GLib, GdkPixbuf, Gio

(pick-colour-picker:6507): Gtk-WARNING **: Failed to parse /etc/gtk-3.0/settings.ini: Permission denied

(pick-colour-picker:6507): GdkPixbuf-WARNING **: Cannot open pixbuf loader module file '/home/aquarius/snap/pick-colour-picker/x1/.cache/gdk-pixbuf-loaders.cache': No such file or directory

This likely means that your installation is broken.
Try running the command
  gdk-pixbuf-query-loaders > /home/aquarius/snap/pick-colour-picker/x1/.cache/gdk-pixbuf-loaders.cache
to make things work again for the time being.
Gtk-Message: Failed to load module "canberra-gtk-module"
Traceback (most recent call last):
  File "/snap/pick-colour-picker/x1/lib/python2.7/site-packages/pick/__main__.py", line 713, in handle_commandline
    self.start_everything_first_time()
  File "/snap/pick-colour-picker/x1/lib/python2.7/site-packages/pick/__main__.py", line 868, in start_everything_first_time
    self.w.set_default_icon(theme_icon.load_icon())
GLib.Error: gdk-pixbuf-error-quark: Unrecognized image file format (3)
```

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -1,0 +1,45 @@
+name: pick-colour-picker
+version: 'master'
+summary: A colour picker that remembers where you picked colours from
+description: |
+  Choose the colour you want and Pick remembers it, names it, and shows you a screenshot
+  so you can remember where you got it from. Zoom all the way in to pixels to pick
+  just the right one. Show your colours in your choice of format: rgba() or hex,
+  CSS or Gdk or Qt, whichever you prefer. Copy to the clipboard ready for pasting into
+  code or graphics apps.
+icon: pick-colour-picker.png
+
+grade: devel
+confinement: strict
+
+apps:
+  pick-colour-picker:
+    command: desktop-launch pick-colour-picker
+    desktop: usr/share/applications/pick-colour-picker.desktop
+    plugs:
+      - x11
+      - pick-colour-picker-dbus
+
+plugs:
+  pick-colour-picker-dbus:
+    interface: dbus
+    name: org.kryogenix.pick
+    bus: session
+
+parts:
+  pick-colour-picker:
+    source: .
+    plugin: python
+    python-version: python2
+    build-packages:
+      - libgirepository1.0-dev
+    stage-packages:
+      - python-gi
+      - python-gobject
+      - libgirepository1.0-dev
+      - python-cairo
+      - python-gi-cairo
+      - gir1.2-gtk-3.0
+    after: [desktop-gtk3]
+    prepare: |
+      sed -i 's|Icon=pick-colour-picker|Icon=${SNAP}/usr/share/pixmaps/pick-colour-picker.png|g' pick-colour-picker.desktop

